chainbase/src/main/java/org/tron/core/capsule/BlockCapsule.java

```diff
@@ -314,7 +313,7 @@ public boolean hasWitnessSignature() {
 
   @Override
   public String toString() {
-    toStringBuff.setLength(0);
+    StringBuilder toStringBuff = new StringBuilder();
```

Comment:
why not replace the origin field with StringBuffer?

Reply:
@lurais toStringBuff is only used in toString() method, using local variable makes code easier to understand.